libr/anal/var.c

```diff
@@ -1092,12 +1092,12 @@ R_API RList *r_anal_function_get_var_fields(RAnalFunction *fcn, int kind) {
 
 static int var_comparator(const RAnalVar *a, const RAnalVar *b){
 	// avoid NULL dereference
-	return (a && b)? a->delta > b->delta: false;
+	return (a && b)? (a->delta > b->delta) - (a->delta < b->delta) : 0;
```

Comment:
Why not:
```suggestion
	return (a && b)? (a->delta - b->delta) : 0;
```
?

Reply:
@ret2libc What is the expected value range for delta and argnum? Do we know that they will never exceed half of integer range?

Reply:
yeah good point actually... maybe it is ok as it was, sorry for the mistake ;)